packages/metro/src/start/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './start';
+export * from './runServer';
```

Comment:
Again, `index.js` file is not needed here in my humble opinion (sticking to the other plugins in the CLI).

Right now, since `runServer` and `getReporterImpl` are exported from `runServer`, they are also exported to the public (everything from `start` is exported to the public via `src/index.js`).

While these two functions are used internally in other files, **I wouldn't change any exports in this PR beyond the single one that you have requested.** 

For example, `getReporterImpl` is not needed anywhere outside of `runServer.ts` file - there's no need to even export it internally, not mentioning exporting it as a public interface.

Reply:
Got rid of it and limited the exports to only what is needed for the cli project.